core/src/main/scala/org/apache/spark/MapOutputTracker.scala

```diff
@@ -629,6 +645,35 @@ private[spark] class MapOutputTrackerMaster(
     None
   }
 
+  /**
+   * Return the locations where the Mapper(s) ran. The locations each includes both a host and an
+   * executor id on that host.
+   *
+   * @param dep shuffle dependency object
+   * @param startMapId the start map id
+   * @param endMapId the end map id
+   * @return a sequence of locations that each includes both a host and an executor id on that
```

Comment:
`includes both a host and an executor id` is confusing. We can just say `task location strinng (please refer to TaskLocation)`